crates/bevy_ui/src/stack.rs

```diff
@@ -35,7 +35,7 @@ impl ChildBufferCache {
 
 /// Generates the render stack for UI nodes.
 ///
-/// Create a list of root nodes from unparented entities and entities with a `GlobalZIndex` component.
+/// Create a list of root nodes from parentless entities and entities with a `GlobalZIndex` component.
```

Comment:
We are using `parentless` in similar contexts:

https://github.com/bevyengine/bevy/blob/0707c0717b6a3156a890d8800b84d746a6c29bfd/crates/bevy_transform/src/commands.rs#L89

https://github.com/bevyengine/bevy/blob/0707c0717b6a3156a890d8800b84d746a6c29bfd/crates/bevy_ui/src/layout/ui_surface.rs#L19-L20